### DISCUSSION TITLE
Early Access Hub vs. Demo vs. Hubs Cloud

### DISCUSSION THREAD

**Discussion by jalgrim** [mozilla/hubs#5940] [Hubs Cloud Questions] [answered]
Hi, as I look through the Mozilla Hubs site I am confused at what is being offered.  There is the early access Hub for $20/month which boasts features I seem to already be getting for the Demo version.  What do I get for $20/month that I am not already getting in my demo?  It seems I already can make lots of spaces and have up to 25 participants in a space--what exactly are the limits on the demo version?

Does Early Access give me a webpage something like [this](https://hubs.mozilla.com/demo) that can showcase all my own spaces?  Whereas the demo version allows me to create spaces, but with no easy place to showcase them?

Then there is the Hubs cloud which seems to just be a more technical subscription compared to Early Access Hubs that allows more branding?  Is the main difference between Early Access and Hubs Cloud just that you can use your own web address and stylize the site a bit more with Hubs Cloud?

**Comment by pattersonbl2** [comment, marked as answer]
EA is a managed service for running a hub instance.  Currently, it allows you to create a hub instance without having to worry about managing the infrastructure like Hub cloud for AWS.  We will be releasing more features to this offering that will not be in demo space or hubs cloud on AWS.  

The demo version is currently a beta version of the EA. This means the reliability of running an instance isn't ensured. Also, changes will be coming to this version as well that will have certain limitations. We haven't fully outlined the limitation currently


You have correct in your thinking about the differences between hubs cloud and EA.   I would add that we are currently developing similar features for stylizing an EA instance.  We don't have an official timeline but work is currently underway.  Hub cloud also will reach a point where new features are not pushed to the code base for it.

**Reply by jalgrim** [reply]
Thanks @Pattersonbl2 !  I wasn't sure where I could ask this question, and then later found a similar thread on Discord.  Here's the link in case anyone else comes here with this question: [https://discord.com/channels/498741086295031808/1072562937652326541](https://discord.com/channels/498741086295031808/1072562937652326541)
